color/README.md

```diff
@@ -26,6 +26,46 @@ See https://linebender.org/blog/doc-include/ for related discussion. -->
 
 Color is a Rust crate which implements color space conversions, targeting at least CSS4 color.
```

Comment:
This sentence was a stub added to help make the README. Maybe it's optimal as-is, but I would like affirmitive confirmation of that whilst we're here.

Reply:
As a minimal change, I would write "CSS Color Module Level 4", "CSS Color Level 4", or "CSS Color 4" instead of "CSS4 color", to stick closer to the official name – and link that to `https://www.w3.org/TR/css-color-4/`.